Amazon.IonObjectMapper.Test/IonObjectSerializerTest.cs

```diff
@@ -619,5 +619,77 @@ private void AssertIsTruck(object actual)
         {
             Assert.AreEqual(actual.ToString(), TestObjects.nativeTruck.ToString());
         }
+
+        [TestMethod]
+        public void TestSerializingAndDeserializingPropertiesWithAccessModifiers()
+        {
+            var classWithProperties = new ClassWithProperties(PublicProperty: "PublicProperty", DefaultProperty: "DefaultProperty",
+                ProtectedProperty: "ProtectedProperty", ProtectedInternalProperty: "ProtectedInternalProperty", InternalProperty: "InternalProperty",
+                PrivateProperty: "PrivateProperty", PrivateProtectedProperty: "PrivateProtectedProperty");
+
+            var deserializedObject = Serde(classWithProperties);
+
+            Assert.AreEqual("PublicProperty", deserializedObject.PublicProperty);
+            Assert.AreEqual("", deserializedObject.GetDefaultProperty());
```

Comment:
Why not do an `Assert.IsNull` and let `GetDefaultProperty` return the property rather than the string version of the property?

Reply:
Updated to `Assert.IsNull`